workspace/AATAMS_ACOUSTIC_QC/Jenkinsfile

```
@@ -0,0 +1,67 @@
+#!groovy​
+
+pipeline {
+    agent none
+    environment {
+        JOB_WORKSPACE = 'workspace/AATAMS_ACOUSTIC_QC'
+        JOB_NAME = 'AATAMS_ACOUSTIC_QC'
+    }
+    stages {
+        stage('validation') {
+            agent { label 'master' }
+            steps {
+                //Moving in to the directory to execute the commands
+                dir(${JOB_WORKSPACE}) {
```

The ${} syntax is only for within strings, it won't work here.
```suggestion
                dir(JOB_WORKSPACE) {
```

---

Also, pretty sure it's exposed in the groovy scope as `env.JOB_WORKSPACE`